ci-operator/config/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.10.yaml

```diff
@@ -162,6 +162,8 @@ tests:
   skip_if_only_changed: \.md$|^(DCO|LICENSE|OWNERS|PROJECT)$
```

Comment:
As per the doc, we should not update `openshift-priv` org: https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization

Reply:
Yes, `make jobs` forced me to. I think some changes to some repos require manual interfere or the cli tool is broken. Anyway the docs say that any manual change will be overridden, so no harm done